src/samples/coap/lwm2m-client.c

```diff
@@ -262,8 +273,7 @@ read_security_server_obj(void *instance_data, void *user_data,
     switch (res_id) {
     case SECURITY_SERVER_SERVER_URI_RES_ID:
         SOL_LWM2M_RESOURCE_INIT(r, res, 0, 1,
-            SOL_LWM2M_RESOURCE_DATA_TYPE_STRING,
-            sol_str_slice_from_str("coap://localhost:5683"));
+            SOL_LWM2M_RESOURCE_DATA_TYPE_STRING, &addr);
```

Comment:
I am assuming you are using blob instead of str_slice because of memory control (ref/unref), right?
In this case, can't you do a helper to add a string literal? Because it is a bit annoying to have to create a blob in all string usages